src/components/forms/form-group/FormGroupCtrl.ts

```diff
@@ -24,17 +24,45 @@ export class FormGroupCtrl {
 
   type: string;
   multiValue: boolean;
+  multiSelect: boolean;
+  multiSettings: any;
   option: string;
   parentId: Function = generateFieldId;
 
   constructor() {
+    this.multiSettings = {
+      template: "{{option}}",
+      smartButtonTextConverter(skip: any, option: any): any {
+        return option;
+      }
+    };
+    this.type = getTypeModelType(this.meta);
+    if (this.meta.type.TYPE_MODEL_VALUE === "LIST") {
```

Comment:
@ryanemerson Yes, exactly that. What do you mean DMR output? When I first loaded the model values into the field it sliced them into one item per character. So the dropdown menu had 20+ items. After I used JSON.parse it all started to work, both model loading and saving.